src/tests/plugins/analytics/AnalyticsTrackerAdapter.test.js

```diff
@@ -5,13 +5,13 @@ const fixtures = {
   application: 'console-kit'
 }
 
-const makeSut = () => {
+const makeSut = (segmentToken = 'VITE_STAGE_SEGMENT_TOKEN') => {
```

Comment:
Não estamos mais usando stage ou prod certo?
```suggestion
const makeSut = (segmentToken = 'VITE_SEGMENT_TOKEN') => {
```

Reply:
done